src/client/Home/NodeView.jsx

```diff
@@ -31,13 +31,11 @@ const NodeView = ({ onRestart, onShutdown }) => {
 
   const engineName = newConfig?.engine?.engineName ?? ''
   const safeMode = newConfig?.engine?.safeMode ?? false
-  const applications = newConfig?.north?.filter(
-    (application) => application.name.toLowerCase().includes(northFilter.toLowerCase()),
-  ) ?? []
-  const dataSources = newConfig?.south?.filter((dataSource) => dataSource.name.toLowerCase().includes(southFilter.toLowerCase())) ?? []
-  const globalZIndex = dataSources.length + applications.length + 1
-  const northNodes = applications.map((application, indexNorth) => ({
-    id: application.id,
+  const northConnectors = newConfig?.north?.filter((north) => north.name.toLowerCase().includes(northFilter.toLowerCase())) ?? []
+  const southConnectors = newConfig?.south?.filter((south) => south.name.toLowerCase().includes(southFilter.toLowerCase())) ?? []
```

Comment:
i havent read the whole thing but why this can be null (i don't like too much abusing with ? when not necessary)